src/pages/projects/actions/DeleteProjectBtn.tsx

```diff
@@ -52,9 +51,14 @@ const DeleteProjectBtn: FC<Props> = ({ project }) => {
           and can result in data loss.
         </>
       }
-      posButtonLabel="Delete"
+      posButtonLabel={
+        project.name === "default"
+          ? "The default project can't be deleted"
+          : "Delete"
+      }
       onConfirm={handleDelete}
       isDense={false}
+      hasShiftHint={false}
```

Comment:
Design requested to not show the hint in this particular case. You can't delete a project if it has anything in it, so I see no reason to avoid the shift-skip.

Reply:
At this point you could rather say "You can't delete a project if it has anything in it, so I see no reason to show a confirmation dialog at all". Anyway, my point was different: we have a hidden feature here. We can skip the confirmation with SHIFT + click but we don't inform the user that this is possible.